doc/devel/documenting_mpl.rst

```diff
@@ -380,26 +380,28 @@ An example docstring looks like:
 
         xmin, xmax : float or array-like
             Respective beginning and end of each line. If scalars are
-            provided, all lines will have the same length.
+            provided, all lines will have same length.
```

Comment:
The 'the' is correct; can you fix the `hlines` docstring instead?

Reply:
Sure!

Reply:
Done! (and vlines)